src/components/select-project/select-project.component.ts

```diff
@@ -10,7 +10,8 @@ export const selectProject: angular.IComponentOptions = {
     availableProjects: '<?',
     showDivider: '<?',
     hideCreateProject: '<?',
-    hideLabel: '<?'
+    hideLabel: '<?',
+    projectSelectionRequired: '<?'
```

Comment:
Can we use `isRequired` or something shorter?

Reply:
Changed to `isRequired`